tensor2tensor/data_generators/allen_brain_utils.py

```diff
@@ -0,0 +1,484 @@
+# coding=utf-8
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utils. for Allen Brain Atlas dataset, download and subimages."""
+
+import math
+import os
+import shutil
+import tempfile
+
+import numpy as np
+import tensorflow as tf
+
+
+# Gracefully inform users of the need to install problem-specific deps.
+try:
```

Comment:
let's try the function approach that i put in the comment in the other file.
i don't want this error message to show on every invocation of t2t-trainer, etc.
but users need to know that they're missing something if they do try to run this problem.
i think the function approach accomplishes both

Reply:
Ah right, thanks.